Exercise4A.sh

```diff
@@ -0,0 +1,3 @@
+grep -w "female" wages.csv | cut -d , -f1,2 | tr ',' ' ' | sort -n -k2 | uniq >> Wages1.txt
+grep -w "male" wages.csv | cut -d , -f1,2 | tr ',' ' ' | sort -n -k2 | uniq >> Wages1.txt
+
```

Comment:
Alternatively, you can sort by the first column and then the second column with command sort:
sort -k1,1d -k2,2n